api/handlers.go

```diff
@@ -0,0 +1,96 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/ONSdigital/dp-recipe-api/recipe"
+	"github.com/ONSdigital/log.go/log"
+	"github.com/gorilla/mux"
+)
+
+//RecipeListHandler - get all recipes
+// USAGE: curl -X GET http://localhost:22300/recipes
```

Comment:
I would remove usage - this should be obvious from the routes package, also defined in swagger spec